packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -210,6 +210,20 @@ export class Pipeline extends cdk.Construct implements events.IEventRuleTarget {
         ]);
     }
 
+    /**
+     * Get a duplicate of this Pipeline's list of Stages.
+     */
+    public get stages(): Stage[] {
```

Comment:
Is this really needed? (Keep surface area to minimum)

Reply:
Sure. Will remove.